src/Pode.psm1

```diff
@@ -1,38 +1,98 @@
+<#
+.SYNOPSIS
+    Pode PowerShell Module
+
+.DESCRIPTION
+    This module sets up the Pode environment, including
+    localization and loading necessary assemblies and functions.
+
+.PARAMETER UICulture
+    Specifies the culture to be used for localization.
+
+.EXAMPLE
+    Import-Module -Name "Pode" -ArgumentList @{ UICulture = 'ko-KR' }
+    Sets the culture to Korean.
+
+.EXAMPLE
+    Import-Module -Name "Pode"
+    Uses the default culture.
+
+.EXAMPLE
+    Import-Module -Name "Pode" -ArgumentList 'it-SM'
+    Uses the Italian San Marino region culture.
+
+.NOTES
+    This is the entry point for the Pode module.
+#>
+
+param(
+    [string]$UICulture
+)
+
 # root path
 $root = Split-Path -Parent -Path $MyInvocation.MyCommand.Path
+# Import localized messages
+if ([string]::IsNullOrEmpty($UICulture)) {
+    $UICulture = $PsUICulture
+}
+
+#Culture list available here https://azuliadesigns.com/c-sharp-tutorials/list-net-culture-country-codes/
+Import-LocalizedData -BindingVariable tmpMsgtable -BaseDirectory (Join-Path -Path $root -ChildPath 'Locales') -UICulture $UICulture -ErrorAction:SilentlyContinue
+if ($null -eq $tmpMsgtable) {
+    try {
+        Import-LocalizedData -BindingVariable tmpMsgtable -BaseDirectory (Join-Path -Path $root -ChildPath 'Locales') -UICulture 'en' -ErrorAction:Stop
+    }
+    catch {
+        throw
+    }
+}
+
+try {
+    # Create the global msgTable read-only variable
+    New-Variable -Name 'msgTable' -Value $tmpMsgtable -Scope Global -Option ReadOnly -Force
```

Comment:
Could we call the resulting variable `$PodeLocale` instead of `$msgTable`?, that way it fits with `$PodeContext`.

You can also make this script scoped, and either:
* Do the same `Import-LocalizedData` lookup in `Pode.Internal.psm1`, or
* In `New-PodeRunspaceState` you can port the existing `$PodeLocale` variable into the state object for other runspaces by adding the following line where the variables are set (you can see the same being done for PodeContext there as well):
```powershell
(New-Object System.Management.Automation.Runspaces.SessionStateVariableEntry -ArgumentList 'PodeLocale', $PodeLocale, $null),
```

Reply:
sure let me change it